docs/library/media/image.md

```diff
@@ -49,3 +49,8 @@ def image_pil_example():
         rx.image(src=ImageState.image)
     )
 ```
+
+```md alert info
+# Using a cv2 image?
```

Comment:
The title of this should be something closer to `Image only accepts URLs and Pillow Images`

as we don't explicitly say what it can take in anywhere.

Reply:
added